package.json

```diff
@@ -15,29 +17,30 @@
     "sort-keys": "4.1.0",
     "stack-utils": "2.0.5",
     "through2": "2.0.3",
-    "ut-config": "^7.9.3",
     "ut-bus": "^7.62.8",
+    "ut-config": "^7.9.3",
+    "ut-function.merge": "^1.5.6",
     "ut-log": "^6.10.4",
     "ut-port": "^6.43.12",
-    "ut-function.merge": "^1.5.6",
     "uuid": "8.3.2",
     "yaml": "1.10.0"
   },
   "devDependencies": {
     "@cucumber/gherkin": "11.0.0",
-    "inquirer-autocomplete-prompt": "1.2.0",
-    "inquirer": "7.3.3",
     "fuzzy": "0.1.3",
+    "inquirer": "7.3.3",
+    "inquirer-autocomplete-prompt": "1.2.0",
     "tap": "^16.2.0",
     "ut-function.dispatch": "^1.4.1",
     "ut-port-script": "^7.2.0"
   },
   "peerDependencies": {
-    "why-is-node-running": "^2.2.0",
-    "json-schema-to-markdown": "1.1.1"
+    "json-schema-to-markdown": "1.1.1",
+    "why-is-node-running": "^2.2.0"
   },
   "bin": {
-    "ut-doc": "./bin/ut-doc.js"
+    "ut-doc": "./bin/ut-doc.js",
```

Comment:
Let's remove ut-doc from here and also the file.

Reply:
[done](https://github.com/softwaregroup-bg/ut-run/pull/100/commits/ff1dbfb32c4a3b6299cbaea18b11209b82682250)